docs/operations-guide/migrating-from-h2.md

```diff
@@ -1,29 +1,56 @@
-- You must be able to connect to the target Postgres or MySQL/MariaDB database in whatever environment you're running this migration command in. So, if you are attempting to move the data to a cloud database, make sure you can connect to that database.
+The difference between a local installation and a production installation of Metabase is the application database. The application database keeps track of all of your Metabase data: your questions, dashboards, collections, and so on.
 
-### Migrating when using Docker
+Metabase ships with an embedded H2 application database that is not recommended for running Metabase in production. The reason Metabase ships with the H2 database is because we want people to spin up Metabase on their local machine and start playing around with asking questions.
 
-We recommend running the migration outside of Docker. You'll need to copy the H2 file out of the Docker container before migrating. For example, if the container is called metabase, you'd run:
+If you want to run Metabase in production, you'll need to use a production-ready application database to store your application data. You can switch from using the default H2 application database at any time, but if you're planning on running Metabase in production, the sooner you migrate to a production application database, the better. If you keeping running Metabase with the default H2 application database, and you don't regularly back it up, the app db could get corrupted, and you could end up losing all of your questions, dashboards, collections, and other Metabase data.
 
-```
-docker cp metabase:/metabase.db/metabase.db.mv.db ./
-```
+You could also choose to run Metabase on a [Metabase Cloud](/pricing) plan, which takes care of all of this stuff for you.
+
+## Databases we recommend for storing your Metabase application data
+
+- [PostgreSQL](https://www.postgresql.org/). Minimum version: 9.4.
+- [MySQL](https://www.mysql.com/). Minimum version: 5.7.7. Required settings (which are the default): `utf8mb4_unicode_ci` collation, `utf8mb4` character set, and `innodb_large_prefix=ON`.
+- [MariaDB](https://mariadb.org/). Minimum version: 10.2.2. Required settings (which are the default): `utf8mb4_unicode_ci` collation, `utf8mb4` character set, and `innodb_large_prefix=ON`.
+
+Go with whichever database you're familiar with. If you're not familiar with any of these, or not sure which to pick, go with Postgres.
+
+## JAR: How to migrate from H2 to your production application database
+
+Metabase provides a custom migration command for migrating to a new application database. Here's what you'll do:
 
-The above command would copy the database file to the directory you ran the command from. With your database file outside of the container, all you need to do is follow the "How to migrate" steps below.
+- [2. Shutdown your Metabase instance](#2-shutdown-your-metabase-instance)
```

Comment:
Maybe either put a bullet point for number 1 in this list or put the info in number one above the list?